stratum/config/CMakeLists.txt

```diff
@@ -6,23 +6,40 @@
 
 set(STRATUM_TDI_BIN_DIR ${STRATUM_SOURCE_DIR}/stratum/hal/bin/tdi)
 set(STRATUM_BFRT_BIN_DIR ${STRATUM_SOURCE_DIR}/stratum/hal/bin/barefoot)
+set(STRATUM_TOOLS_DIR ${STRATUM_SOURCE_DIR}/tools)
 
 if(DPDK_TARGET)
     install(
         FILES
             ${STRATUM_TDI_BIN_DIR}/dpdk/dpdk_port_config.pb.txt
             ${STRATUM_TDI_BIN_DIR}/dpdk/dpdk_skip_p4.conf
+            ${STRATUM_TOOLS_DIR}/tls/ca.conf
```

Comment:
If i understand correctly, after these changes we can see TLS specific conf and script files will be part of P4CP_INSTALL/sbin/ ?

if yes, in that case we need to modify ACC and Bitbake files to not manually copy. 
Maybe this can be taken later as well.

Reply:
yes that's correct, we can take care of ACC scripts once these changes are merged